src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,51 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final double DEFAULT_VALUE = 10.0;
+    private static final int MULTIPLAYER = 100;
+    private ColorSupplier colorSupplier;
+    private Random random = new Random();
+
+    public FigureSupplier() {
+        this.colorSupplier = new ColorSupplier();
```

Comment:
consider initializing random and colorSupplier in a unified way, either using a constructor or initializing in the fields.